packages/remark-iframes/__tests__/index.js

```diff
@@ -146,57 +200,49 @@ test('video', () => {
 
     !(https://www.youtube.com/watch?v=FdltlrKFr1w)
     with text after
-  `, config)
+  `, config.video)
 
   expect(contents).toMatchSnapshot()
 })
 
-test('extra', () => {
+test('oembed falls back', async () => {
```

Comment:
Added a test to check that oembed properly falls back to a link when it cannot fetch